tree.go

```diff
@@ -611,7 +611,7 @@ walk: // Outer loop for walking the tree
 				if rb[0] != 0 {
 					// Old rune not finished
 					idxc := rb[0]
-					for i, c := range []byte(n.indices) {
+					for i, c := range bytesconv.StringToBytes(n.indices) {
```

Comment:
why add it here, but not add before?

Reply:
loop string type and don't cpompare with a byte don't need to do it, but that loop used so change to slice is necessary, and use StringToBytes will be better. Thanks!

Reply:
thanks. using local var save the result of `StringToBytes`?

Reply:
I believe it's unnecessary to call `bytesconv.StringToBytes` in a `for range` loop, there is a compiler optimization that will avoid the copies, see https://github.com/golang/go/wiki/CompilerOptimizations#range-over-bytes.

Reply:
ok, i see thanks.

Reply:
> thanks. using local var save the result of `StringToBytes`?

that kind of change is not necessary.